in_toto/canonicaljson.go

```diff
@@ -93,7 +93,9 @@ func _encodeCanonical(obj interface{}, result *bytes.Buffer) (err error) {
 			// Note: `key` must be a `string` (see `case map[string]interface{}`) and
 			// canonicalization of strings cannot err out (see `case string`), thus
 			// no error handling is needed here.
-			_encodeCanonical(key, result)
+			if err := _encodeCanonical(key, result); err != nil {
```

Comment:
I think I can remove the error handling in this area, according to the comment, right?

Reply:
removed in newest commit